frontend/projects/upgrade/src/app/core/experiments/store/experiments.model.ts

```diff
@@ -569,6 +569,16 @@ export interface InteractionEffectLineChartSeriesData {
   participantsLogged: number;
 }
 
+export enum EXPERIMENT_ROW_ACTION {
+  EDIT = 'edit',
+  DELETE = 'delete',
+}
+
+export interface ExperimentDecisionPointRowActionEvent {
```

Comment:
Just a clarification here: there are no endpoints to edit individual decision points. All changes made on the experiment details page (besides inclusion/exclusion lists) should be treated as edits to the experiment itself.